vmdb/app/models/ems_refresh/save_inventory_helper.rb

```diff
@@ -1,8 +1,20 @@
 module EmsRefresh::SaveInventoryHelper
   def save_inventory_multi(type, klass, parent, hashes, deletes, find_key, child_keys = [], extra_keys = [])
+    return if hashes.nil?
+    # refresh association if developer is using new syntax (alt: dev passes in nil, [], or actual association values)
+    parent.send(type, true) if deletes == true || deletes == false
```

Comment:
I was thinking about this over the weekend, and I'm not sure I like true/false/[values] as the triple.  I think I prefer the alt from your comment there: nil/[]/[values].